doc/specs/#888 - Com Api.md

```diff
@@ -0,0 +1,812 @@
+
+# 1. Background

```

Comment:
Consider renaming this file from `#888 - Com API.md` to just `winget-com-api.md` or something